installer/Migration/V5_1_0/permission/api.yaml

```diff
@@ -322,6 +322,23 @@ apiv2_recruitment_candidate_shortlist:
     - { role: Admin, permission: { read: false, create: false, update: true, delete: false, self: false } }
     - { role: HiringManager, permission: { read: false, create: false, update: true, delete: false, self: false } }
 
+
+##########################
```

Comment:
I think when you resolving conflict did it wrong. this section should go after `apiv2_recruitment_candidate_mark_interview_failed`